apps/coursition/src/app/page.tsx

```diff
@@ -1,202 +1,188 @@
+            {benefits.map((it) => (
+              <li key={it.text} className='flex flex-col gap-2 items-center max-w-[240px]'>
+                {it.icon}
+                <span className='text-purple-700 text-center font-semibold text-[16px] '>{it.text}</span>
+              </li>
+            ))}
+          </ul>
+
+          <div className='w-full flex items-center justify-center mt-8'>Interactive demo</div>
+        </div>
+      </section>
+
+      <section className='bg-black text-white -mx-12'>
+        <div className='flex items-center justify-center font-semibold text-[24px] h-[160px] text-center px-4'>
+          Maximize the value of video content by making every word searchable.
+        </div>
+      </section>
+
+      <section className='bg-white text-black flex flex-col gap-2 -mx-12 items-center justify-center mt-5'>
+        <h4 className='font-semibold text-[24px]'>Lacking transcription?</h4>
+        <p className='mt-2'>
+          Important meeting recorded, but no way to quickly review key points. =&gt; "What was that action item again?"
+          Cue endless audio scrolling.
+        </p>
+        <p>
+          Brilliant podcast insights, but sharing means manual note-taking. =&gt; Love that podcast gem? Hope you enjoy
+          pausing and typing.
+        </p>
+        <p>
+          Hours of video content, but can't find that crucial quote. =&gt; Needle in a haystack? More like a word in a
+          3-hour video.
+        </p>
+
+        <h4 className='font-semibold text-[24px] mt-4'>Time to unlock your content</h4>
+
+        <BuyLifetime className='mt-4' withIcon />
+      </section>
+
+      <section className='mt-12 py-6 bg-gray-50 text-black flex flex-col gap-4 items-center justify-center -mx-12'>
+        <h4 className='font-semibold text-[24px]'>Why us?</h4>
+        <p className='max-w-[400px] text-[16px]'>
+          Simplify your media. Best AI transcription, under your control. User-friendly interface, flexible pricing.
+          Make videos searchable and AI-ready. For creators, businesses, and consumers.
+        </p>
+
+        <BuyLifetime className='mt-4' withIcon />
+      </section>
+
+      <section className='mt-12 container mx-auto grid grid-cols-1 md:grid-cols-2'>
+        <div className='h-full flex items-center justify-center'>Insert image here</div>
+
+        <div className='flex flex-col gap-4'>
+          <h3 className='font-semibold text-[35px] max-w-[557px] leading-[60px]'>Smart transcription on your terms</h3>
+
+          <div className='w-full h-[150px]' />
+          <BuyLifetime withIcon={false} />
+        </div>
+      </section>
+
+      <section className='mt-12 py-12 bg-gray-50 text-black flex flex-col gap-8 items-center justify-center -mx-12'>
+        <h4 className='font-semibold text-[24px]'>3 simple steps to solve your customer’s problem</h4>
+
+        <ul className='grid grid-cols-1 md:grid-cols-3'>
+          {[
```

Comment:
**nitpick:** As other similar arrays are extracted to the top, this one should be too.